### PR TITLE
feat: Add network filter to top collection

### LINKED ISSUE #8816: Add network filter to top collection
I wish you all the best for the new year 

lets start with one addition I believe will make user less confused about the networks

image

- lets do something like that, but without the "network:" please 

- changing based on the network switch in the navbar 

https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1287-18959&mode=design&t=ICUuvAohcPrUiimF-11

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #8816

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-01-01 at 11 51 24

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

**Comment by exezbcz** [comment]
uwu nice! 


could you please align it

image

**Comment by hassnian** [comment]
@exezbcz pls check again

**Comment by exezbcz** [comment]
last thing, mobile 

image

**Comment by hassnian** [comment]
> last thing, mobile 
> 
> 
> 
> image
> 
> 

can you elaborate? margin missing or/and should be at the end of the row 

thanks

**Comment by exezbcz** [comment]
@hassnian yeah margin under the network button

**Review by preschian — APPROVED** [review]
code lgtm

File: components/landing/topCollections/TopCollections.vue [hunk 1]
```diff
@@ -11,42 +11,48 @@
...
+      <div class="pt-2 pb-5">
+        <ChainDropdown
+          position="bottom-left"
+          :show-network-label="false"
+          :redirect="false"
+          mobile-modal />
+      </div>
     </div>
 
     <div class="top-collections-grid mb-5">
```
Comment (roiLeo) [inline components/landing/topCollections/TopCollections.vue:23]:
```suggestion
    <div class="top-collections-grid my-5">
```

File: components/landing/topCollections/TopCollections.vue [hunk 2]
```diff
@@ -11,42 +11,48 @@
           :label="`${$t(`topCollections.timeFrames.${label}`)}`"
           @click="setTimeRange(value)" />
       </div>
-      <div></div>
+      <div class="pt-2 pb-5">
```
Comment (roiLeo) [inline components/landing/topCollections/TopCollections.vue:14]:
```suggestion
      <div class="pt-2">
```

**Review by roiLeo — APPROVED** [review]
✅ lgtm

**Review by Jarsen136 — APPROVED** [review]

**Comment by exezbcz** [comment]
hmmm, lets remove rmrk2 from the options, the top collection section is not visible on the network 

image

- would be nice if we would have top collections for rmrk2 as well, but not prio

**Comment by exezbcz** [comment]
@roiLeo yup, lets remove it then

**Review by roiLeo — APPROVED** [review]

**Comment by vikiival** [comment]
cc @prury ?

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 7.46 USD/DOT ~ 6.702 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xc0101a84db039c53ef7de9d8269e2045c76170cd42bc11f0d21d3eee9d68e574](https://polkadot.subscan.io/extrinsic/0xc0101a84db039c53ef7de9d8269e2045c76170cd42bc11f0d21d3eee9d68e574)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**